WcaOnRails/app/views/job_failure_mailer/notify_admin_of_job_failure.html.erb

```diff
@@ -0,0 +1,6 @@
+<p>
+  Job <%= @job.id %> failed.
```

Comment:
Ok, this now looks like this:

![image](https://cloud.githubusercontent.com/assets/277474/15365983/304dd170-1cd7-11e6-879b-cc346eafa915.png)